### PR TITLE
wtf should report extensions and other meaningful entrypoints

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

File: datalad/plugin/wtf.py [hunk 1]
```diff
@@ -76,14 +76,24 @@ def __call__(dataset=None, sensitive=None, clipboard=None):
...
         import json
 
+        extractors={}
+        for ep in iter_entry_points('datalad.metadata.extractors'):
+            status = 'BROKEN'
+            try:
+                ep.load()
+                status = 'OK'
+            except:
+                pass
```
Comment (yarikoptic) [inline datalad/plugin/wtf.py]: would have been a good spot to use `exc_str` imho to not leave just noninformative BROKEN

File: datalad/plugin/wtf.py [hunk 2]
```diff
@@ -187,7 +201,8 @@ def _t2s(t):
                      if ds_meta else '[no metadata]'
             ),
             externals=external_versions.dumps(preamble=None, indent='', query=True),
-            metaextractors='\n'.join(p for p in dir(metaextractors) if not p.startswith('_')),
+            extensions='\n'.join(ep.name for ep in iter_entry_points('datalad.extensions')),
+            metaextractors='\n'.join('{}: {}'.format(k, v) for k, v in extractors.items()),
```
Comment (yarikoptic) [inline datalad/plugin/wtf.py]: use `format_dict(extractors)` for uniform appearance/handling?

File: datalad/plugin/wtf.py (hunk 1 above)
Comment (mih) [reply datalad/plugin/wtf.py]: Done

File: datalad/plugin/wtf.py (hunk 2 above)
Comment (mih) [reply datalad/plugin/wtf.py]: Done

**Comment by mih** [comment]
Still a massive timeout issue in one test run: https://travis-ci.org/datalad/datalad/jobs/365441895

Which cannot possibly be related to this change.

**Pull request merged**